### DISCUSSION TITLE
I keep receiving notifications about vulnerabilities on `yarn.lock`, a file which is no longer in my repo

### DISCUSSION THREAD

**Discussion by hamoid** [community/community#39179] [Code Security]
### Select Topic Area

Bug

### Body

Here's the security report:
https://github.com/openrndr/openrndr-guide/security/dependabot/50

It complains about `yarn.lock`. I don't have such a file in my repo.

The alert contains a link to the `yarn.lock` file. If I click it I get a 404.

Anyway I can solve this on my side?

**Comment by david-engelmann** [comment]
[Similar issue with an accepted answer can be found here](https://github.com/orgs/community/discussions/22521)!

**Reply by hamoid** [reply]
Good find :) So the solution is to get help from a developer (GitHub / Dependabot). How to achieve that?

**Comment by david-engelmann** [comment]
[You can submit an issue here](https://github.com/dependabot/dependabot-core). You can also reach out to Github by sending an email to support@github.com.
